braeunig.us data for all bodies other than earth

stratochief66 opened the pull request:
Mars & Venus were already uploaded and merged. I am leaving Earth for @NathanKell as it gave me some trouble.

Integrate the data from: http://www.braeunig.us/KSP/RSS/index.html

stratochief66 committed:
braeunig.us data for all bodies other than earth

Mars & Venus were already uploaded and merged. I am leaving Earth for @NathanKell as it gave me some trouble.

Integrate the data from: http://www.braeunig.us/KSP/RSS/index.html

NathanKell commented on GameData/RealSolarSystem/RSSKopernicus.cfg:
```diff
@@ -896,9 +896,9 @@
-			deactivateAltitude = 130000
-			fadeStart = 80000
-			fadeEnd = 120000
+			deactivateAltitude = 101000
```
@stratochief66 Deactive altitude is also activate altitude. It should be well above the fadeEnd value. or you may get a transparent planet for a frame or two while PQS builds.

NathanKell replied:
Also, the PQS needs to have higher fade values than scaled space. That way scaled space fades in before PQS fades out, and vice versa. Otherwise you can get like 50% opacity planets.

stratochief66 committed:
reverse earth SS - PQS transition data

Glad I caught this. Those are the values I use to work well with RVE/scatterer. They are by no means perfectly tuned for that purpose, but they work. They have the goal of minimizing the transition/fade time since with those graphical mods the transition doesn't look very good.

About the original commit; I commented out the sections I believe @NathanKell wanted commented out, but present for future use.

NathanKell commented:
Merging this in as is, will tweak the fades. Thanks again! :)
